src/components/application_manager/test/commands/mobile/on_system_request_notification_test.cc

```diff
@@ -122,7 +122,7 @@ bool CheckHeader(MessageSharedPtr msg) {
   required_members.insert("ContentType");
   required_members.insert("ConnectTimeout");
   required_members.insert("DoOutput");
-  required_members.insert("Content-Length");
+  required_members.insert("Content_Length");
```

Comment:
@okozlovlux please investigate https://adc.luxoft.com/jira/browse/APPLINK-27238, I am not sure that is is correct change.